src/theme-preval.ts

```diff
@@ -348,11 +349,16 @@ const stateLabels = {
   }
 }
 
+const primerPrimitivesColors = splitCompoundColor(primitives.colors.light)
```

Comment:
[minor] I might call this variable `functionalColors`:

```suggestion
const functionalColors = splitCompoundColor(primitives.colors.light)
```